test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/Discovery/DiscoveryManagerTests.cs

```diff
@@ -221,6 +226,25 @@ public void DiscoverTestsShouldCollectMetrics()
             mockMetricsCollector.Verify(rd => rd.Add(TelemetryDataConstants.TotalTestsDiscovered, It.IsAny<object>()), Times.Once);
         }
 
+        [TestMethod]
+        public void DiscoveryInitializeShouldVerifyWarningMessageIfAdapterFailedToLoad()
+        {
+            var assemblyLocation = typeof(DiscoveryManagerTests).GetTypeInfo().Assembly.Location;
+            var mockLogger = new Mock<ITestDiscoveryEventsHandler2>();
+            TestPluginCacheTests.SetupMockExtensions(
+                new string[] { assemblyLocation },
+                () => { });
+
+            //Act
+            this.discoveryManager.Initialize(new List<string> { assemblyLocation }, mockLogger.Object);
```

Comment:
>assemblyLocation [](start = 64, length = 16)

make the message as appropriate

Reply:
Removed unnecessary test and updated message.